cmake/ev-project-bootstrap.cmake

```
@@ -5,7 +5,7 @@ include(${CMAKE_CURRENT_LIST_DIR}/config-run-script.cmake)
 include(${CMAKE_CURRENT_LIST_DIR}/config-run-nodered-script.cmake)
 
 # dependencies
-require_ev_cli_version("0.0.22")
+require_ev_cli_version("0.0.23")
```

I guess you could already bump it to 0.0.24 here since that's what ev-cli was bumped to when introducing this feature
```suggestion
require_ev_cli_version("0.0.24")
```